src/Onboarding/EulaModal.tsx

```diff
@@ -1,26 +1,24 @@
-import React, { useEffect, useState, FunctionComponent } from "react"
+import React, { useState, FunctionComponent } from "react"
 import {
   TouchableOpacity,
   Linking,
   Modal,
+  ActivityIndicator,
   StyleSheet,
   View,
   SafeAreaView,
   StatusBar,
   Image,
 } from "react-native"
 import { useTranslation } from "react-i18next"
-import loadLocalResource from "react-native-local-resource"
```

Comment:
Oooo maybe we can delete one dependency here 🤞  Do we know if it's used anywhere else?